testing/manual/transform-dataset.md

```diff
@@ -1,23 +1,33 @@
-## Transform Dataset Operator
-Please go through __every__ step of the test scenario.\
-Report any issues into GitHub: [open an issue](https://github.com/DARPA-ASKEM/terarium/issues/new?assignees=&labels=bug%2C+Q%26A&template=qa-issue.md&title=%5BBUG%5D%3A+).
+Please go through **every** step of the test scenario.  
+Report any issues into GitHub: [open an issue](https://github.com/DARPA-ASKEM/terarium/issues/new?assignees=&labels=bug%2C+Q%26A&template=qa-issue.md&title=%5BBUG%5D%3A+).
```

Comment:
Why is `transform-dataset.md` here? It is weirdly out of place - if you are making testing notes changes let's put that into a different PR to make things easier to track.

Reply:
It looks like this stated showing up in multiple people's PR lol